gef.py

```diff
@@ -220,19 +223,19 @@ def highlight_text(text):
     return "".join(ansiSplit)
 
 
-def gef_print(x="", *args, **kwargs):
+def gef_print(x: str = "", *args: Tuple, **kwargs: Dict) -> Optional[int]:
     """Wrapper around print(), using string buffering feature."""
     x = highlight_text(x)
     if gef.ui.stream_buffer and not is_debug():
         return gef.ui.stream_buffer.write(x + kwargs.get("end", "\n"))
     return print(x, *args, **kwargs)
```

Comment:
weird. Why do we return print here? Mind changing to print then return? This whole function can probably just return None, actually.

Reply:
I actually encountered quite some similar questionable practices along the way ;)

In regards to changing this code in this PR: If I were to start changing things so it makes more sense this PR will become even huger and very confusing. IMO code changes should be implemented in separate PRs.

Reply:
which is fine, but we will forget unless we note this done somewhere. Maybe make another issue and just add notes from this PR for things to fix?

Reply:
I opened a project for this at https://github.com/hugsy/gef/projects/11